code/modules/mob/living/carbon/xenomorph/abilities/shrieker/shrieker_abilities.dm

```diff
@@ -0,0 +1,110 @@
+/datum/action/xeno_action/activable/shriek
+	name = "Shriek"
+	action_icon_state = "screech"
```

Comment:
needs a diff sprite imo

Reply:
naw screech is a general icon, kind of like pounce is for lurker/runner/ravager.